pkg/asset/kubeconfig/kubeconfig.go

```diff
@@ -0,0 +1,157 @@
+package kubeconfig
+
+import (
+	"encoding/base64"
+	"fmt"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/types"
+	clientcmd "k8s.io/client-go/tools/clientcmd/api/v1"
+)
+
+const (
+	// KubeconfigUserNameAdmin is the user name of the admin kubeconfig.
+	KubeconfigUserNameAdmin = "admin"
+	// KubeconfigUserNamekubelet is the user name of the kubelet kubeconfig.
+	KubeconfigUserNamekubelet = "kubelet"
```

Comment:
nit: `Namekubelet` -> `NameKublet`.

Reply:
* `NameKubelet`

Reply:
I think it should be kubelet https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

Reply:
I see